libraries/CurieSoftwareSerial/SoftwareSerial.cpp

```diff
@@ -94,14 +94,21 @@ bool SoftwareSerial::listen()
     bufferOverflow = false;
     _receive_buffer_head = _receive_buffer_tail = 0;
     active_object = this;
-    rxPin = _receivePin;
+    _rxPin = _receivePin;
+    rxIntraBitDelay = _rx_delay_intrabit;
+    rxCenteringDelay = _rx_delay_centering;
```

Comment:
Is there a good reason for defining any global variables here? This essentially makes all of these variables "reserved words" for anyone who includes this library in their sketch.